store/conf/attrs/zimbra-attrs.xml

```diff
@@ -3720,7 +3720,7 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
 
 <attr id="704" name="zimbraFeatureMailForwardingInFiltersEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable" since="5.0.10">
   <defaultCOSValue>TRUE</defaultCOSValue>
-  <desc>enable end-user mail forwarding defined in mail filters features</desc>
+  <desc>enable end-user mail redirecting defined in mail filters features</desc>
```

Comment:
@rupeshtelus this being LDAP change and the bug affected version is 8.8.x, this change won't make update for older version as in the patch release LDAP schema won't be updated. Considering this do we need this change?

Reply:
@dasiyogesh as I had a discussion with @dawoodshaikh , he is ok to validate this LDAP changes in 9.1.0 Release